### PR TITLE
Rework logging

### PR DESCRIPTION
**Pull request by RobinSikkens**
Rework logging to output to both console and `/logs/bot.log`

use:
```python
import logging

LOGGER = logging.getLogger(__name__)
LOGGER.info("This gets logged")
```

Note that only INFO and higher messages get posted to console DEBUG+ get logged in bot.log

### CODE REVIEW

**Review by romanpeters — APPROVED** [review]
Looks good. Does it work now?

**Comment by romanpeters** [comment]
Optional environment variables to adjust the logging levels would be useful.

File: README.md [hunk 1]
```diff
@@ -66,3 +66,13 @@ async def echo_command(message_contents, *_args, **_kwargs):
     """ On !echo string or !repeat string replies with string """
     return ' '.join(message_contents)
 ```
+
+## Supported optional `.env` tokens
```
Comment (romanpeters) [inline README.md]: tokens → variables

File: devbot/bot.py [hunk 2]
```diff
@@ -68,24 +71,28 @@ def main():
...
 def logging_setup():
+    file_level = logging.DEBUG
+    console_level = logging.INFO
+    if "FILE_LOGLEVEL" in os.environ.keys():
+        file_level = os.environ["FILE_LOGLEVEL"]
+    if "CONSOLE_LOGLEVEL" in os.environ.keys():
+        console_level = os.environ["CONSOLE_LOGLEVEL"]
     # Set up basic functions to log to a file
-    logging.basicConfig(level=logging.DEBUG,
+    logging.basicConfig(level=file_level,
```
Comment (romanpeters) [inline devbot/bot.py]: This doesn't seem right. Does `logging.basicConfig()` accept `level=logging.DEBUG` and `level="DEBUG"` interchangeably?

**Review by romanpeters — APPROVED** [review]

File: devbot/bot.py (hunk 2 above)
Comment (RobinSikkens) [reply devbot/bot.py]: Yes, I've tested it and it works

**Pull request merged**